sunscreen_runtime/src/runtime.rs

```diff
@@ -258,15 +262,19 @@ where
      * Validates and runs the given FHE program. Unless you can guarantee your FHE program is valid,
      * you should use this method rather than [`run_program_unchecked`].
      */
-    pub fn run<I>(
+    fn run_impl<I>(
         &self,
         fhe_program: &CompiledFheProgram,
         mut arguments: Vec<I>,
         public_key: &PublicKey,
+        dbg_info: Option<DebugInfo>,
+        #[cfg(feature = "debugger")]
+        source_code: &str
     ) -> Result<Vec<Ciphertext>>
     where
         I: Into<FheProgramInput>,
     {
+        println!("{:?}", "run_program_unchecked".to_owned());
```

Comment:
Remove any println! statements you have. If this is useful for end users (not just you) to debug their code, bring in the `log` crate as a dependency and call `trace!`.

Users can then use a logger with the level set to trace and see these statements.